### ISSUE TITLE
Proyecto 01

### ISSUE THREAD

**Issue by domingomery** [domingomery/vision#4]
Por favor, escribir aquí comentarios y hacer preguntas (siempre y cuando no se hayan hecho en clases). Gracias!

**Comment by isinahum** [comment]
Hola! 

Para el paso 4, ¿es válido tomar m1 como uno de los puntos que utilizamos en el paso 2, o debe ser un punto "nuevo"? Y otra pregunta, ¿el punto m2 se determina de forma empírica (con alguna forma que hayamos visto en clases) o podemos definirlo a mano?

Estaré atenta, saludos y gracias de antemano!

**Comment by domingomery** [comment]
> Para el paso 4, ¿es válido tomar m1 como uno de los puntos que utilizamos en el paso 2, o debe ser un punto "nuevo"?

No es valido, es necesario usar un punto nuevo que no haya sido usado en la calibración


> ¿el punto m2 se determina de forma empírica (con alguna forma que hayamos visto en clases) o podemos definirlo a mano?

Esto lo dije en la clase del 26/09, ver video en youtube.

**Comment by domingomery** [comment]
> En pasos adicionales, a que se refiere con incluir mejoras al método propuesto?

Cómo se puede mejorar este método? para qué otra función puede servir lo que han hecho? algo adicional que se les ocurra? el proyecto consiste en medir la altura de un objeto, podría servir para otra cosa? aquí la idea es que sugieran pasos adicionales (o alternativos) y que los implementen.

**Comment by Incruen** [comment]
Hola!

¿Está permitido hacer una sección de "Anexo" en el informe para agregar imágenes con extractos de código y resultados, cuyas páginas no cuente dentro del límite de 8 hojas?

**Comment by domingomery** [comment]
> Está permitido hacer una sección de "Anexo" en el informe para agregar imágenes con extractos de código y resultados, cuyas páginas no cuente dentro del límite de 8 hojas?

No. El informe no debe superar las 8 páginas en total. Usen el Colab para poner las imágenes que quieran.
